### ISSUE TITLE
[false positive] goto.walmart.com

### ISSUE THREAD

**Issue by dnmTX** [lightswitch05/hosts#280]
@lightswitch05 apparently i can't open external links with that domain blocked. For example,go to:
https://brickseek.com/walmart-inventory-checker?sku=845991532  and click on **View on Walmart**:
![BrickSeek](https://user-images.githubusercontent.com/34774426/119053826-3cb53280-b98c-11eb-954a-49faadacd45f.png)

Does it track or something? It's kind of inconvenient not to be able to reach the main website right?
Let me know,thanks 👍

**Comment by llacb47** [comment]
It's an affiliate link fyi

**Comment by dnmTX** [comment]
> It's an affiliate link fyi

Well,bunch of sites have those and blocking them links makes it worse,not better.
That's why i much rather have them not blocked and just use the [proper extension](https://chrome.google.com/webstore/detail/skip-redirect/jaoafjdoijdconemdmodhbfpianehlon) which bypassing them altogether.

**Comment by lightswitch05** [comment]
Thanks for the report @dnmTX. Also, I appreciate the added information @llacb47 - I see you got this domain added to your `mischosts: Blacklist` a couple days before I added it to mine.

Unfortunately I failed to capture great notes on this one. This is what I have:

> seems like it should be blocked by reading https://impact.com/

As far as I can tell from the link you provided @dnmTX - it seems to be a link-only domain and not loading up scripts or anything else from it. I personally don't like affiliate marketing, but if someone chooses to click on something, then I certainly don't want to break it. In this case it does appear to be links only. So I'm going to remove it. If it comes back up and appears to be loading scripts rather then just links - then I might add it back. It will be removed in the next update tonight.

Thank you everyone for the great discourse! Have a great weekend!

https://www.youtube.com/watch?v=fIERyg5dYZw
